packages/components/src/logos/wordpress-logo/index.tsx

```diff
@@ -17,7 +18,10 @@ type Props = {
 };
 
 export const WordPressLogo: React.FunctionComponent< Props > = ( {
-	className = 'wordpress-logo',
+	// TODO: This behavior of the `className` prop overriding the default classes are not standard,
+	// and should be normalized to be merged instead. Ideally, we also remove the `wordpress-logo`
+	// class because it clashes very easily.
```

Comment:
>Ideally, we also remove the `wordpress-logo` class because it clashes very easily.

With the eventual removal of `wordpress-logo`, what's the desired end-state where a component is currently overriding styles of `.wordpress-logo` ? Would that component be expected to pass its own `className` to reference it in its styles?

Reply:
Removing the hardcoded `.wordpress-logo` would be a breaking change, although I'm ok with it if communicated well.

> Would that component be expected to pass its own className to reference it in its styles?

Yes, that's my expectation